getusermedia.html

```diff
@@ -1640,21 +1640,28 @@ <h2>Constrainable Properties</h2>
               <td>{{ConstrainULong}}</td>
               <td>The width or width range, in pixels. As a capability, the
               range should span the video source's pre-set width values with
-              min being the smallest width and max being the largest
-              width.</td>
+              min being equal to 1 and max being the largest
+              width. The User Agent MUST support downsampling to any value
+              between the min width range value and the native resolution width.</td>
             </tr>
             <tr id="def-constraint-height">
               <td><dfn>height</dfn></td>
               <td>{{ConstrainULong}}</td>
               <td>The height or height range, in pixels. As a capability, the
               range should span the video source's pre-set height values with
-              min being the smallest height and max being the largest
-              height.</td>
+              min being equal to 1 and max being the largest
+              height. The User Agent MUST support downsampling to any value
+              between the min height range value and the native resolution height.</td>
             </tr>
             <tr id="def-constraint-frameRate">
               <td><dfn>frameRate</dfn></td>
               <td>{{ConstrainDouble}}</td>
               <td>The exact frame rate (frames per second) or frame rate range.
+              If video source's pre-set can determine frame rate values, the range, as a capacity,
+              should span the video source's pre-set frame rate values with
+              min being equal to 1 and max being the largest frame rate.
```

Comment:
Need to allow frame durations larger than 1 second. How about
```suggestion
              min being equal to 0 and max being the largest frame rate.
```